wolfcrypt/_build_ffi.py

```diff
@@ -381,11 +385,53 @@ def generate_libwolfssl():
                             RsaKey* key);
     int wc_RsaPublicEncrypt(const byte*, word32, byte*, word32,
                             RsaKey*, WC_RNG*);
-
-    int wc_RsaSSL_Sign(const byte*, word32, byte*, word32, RsaKey*, WC_RNG*);
-    int wc_RsaSSL_Verify(const byte*, word32, byte*, word32, RsaKey*);
+    int wc_RsaPublicEncrypt_ex(const byte* in, word32 inLen, byte* out,
+               word32 outLen, RsaKey* key, WC_RNG* rng, int type,
+               enum wc_HashType hash, int mgf, byte* label, word32 labelSz);
+    int wc_RsaPrivateDecrypt_ex(const byte* in, word32 inLen,
+               byte* out, word32 outLen, RsaKey* key, int type,
+               enum wc_HashType hash, int mgf, byte* label, word32 labelSz);
     """
-
+    if RSA_PSS_ENABLED:
+        _cdef += """
+        static const int WC_RSA_PKCSV15_PAD;
+        static const int WC_RSA_OAEP_PAD;
+        static const int WC_RSA_PSS_PAD;
+        static const int WC_RSA_NO_PAD;
+
+        static const int WC_MGF1NONE;
+        static const int WC_MGF1SHA1;
+        static const int WC_MGF1SHA224;
+        static const int WC_MGF1SHA256;
+        static const int WC_MGF1SHA384;
+        static const int WC_MGF1SHA512;
+
+        static const int WC_HASH_TYPE_NONE;
```

Comment:
Do all of the constants here need to be gated on `RSA_PSS_ENABLED`? I think many of these can be added unconditionally.

Reply:
Technically they all could. I'll move them out.